Test fail with combination of xml2-config and pkg-config

Received a test fail report in closed issue here for macOS 10.15.7
https://github.com/PerlAlien/Alien-Libxml2/issues/11#issuecomment-713658647

My mac is currently 10.15.6, and it is working for me.  The cflags I get are different from what is reported.

```
❌ helix% xml2-config --cflags
-I/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include
```

```
Running make test for PLICEASE/Alien-Libxml2-0.16.tar.gz
"/Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/bin/perl" -MAlien::Build::MM=cmd -e test
Alien::Build::Plugin::PkgConfig::Negotiate> Using PkgConfig plugin: PkgConfig::CommandLine
PERL_DL_NONLAZY=1 "/Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
t/00_diag.t ........ 1/? #
#
#
# HARNESS_ACTIVE=1
# HARNESS_VERSION=3.42
# LANG=en_GB.UTF-8
# LC_TERMINAL=iTerm2
# LC_TERMINAL_VERSION=3.3.12
# PERL5LIB=/Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-9/blib/lib:/Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-9/blib/arch:
# PERL5OPT=
# PERL5_CPANPLUS_IS_RUNNING=86928
# PERL5_CPAN_IS_RUNNING=86928
# PERLBREW_HOME=/Users/michaelgrundie/.perlbrew
# PERLBREW_MANPATH=/Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/man
# PERLBREW_PATH=/Users/michaelgrundie/perl5/perlbrew/bin:/Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/bin
# PERLBREW_PERL=perl-5.16.0
# PERLBREW_ROOT=/Users/michaelgrundie/perl5/perlbrew
# PERLBREW_SHELLRC_VERSION=0.88
# PERLBREW_VERSION=0.88
# PERL_DL_NONLAZY=1
# PERL_MM_USE_DEFAULT=1
# PERL_USE_UNSAFE_INC=1
# SHELL=/bin/zsh
#
#
#
# PERL5LIB path
# /Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-9/blib/lib
# /Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-9/blib/arch
#
#
#
# perl                                     5.016000
# Alien::Base                              2.33
# Alien::Build                             2.33
# Alien::Build::MM                         2.33
# Alien::Build::Plugin::Build::SearchDep   2.33
# Alien::Build::Plugin::Prefer::BadVersion 2.33
# Alien::Build::Plugin::Probe::Vcpkg       2.33
# ExtUtils::CBuilder                       0.280206
# ExtUtils::MakeMaker                      7.48
# Test2::V0                                0.000136
# Test::Alien                              2.33
#
#
#
# version        = 2.9.4
# cflags         =
# cflags_static  =
# libs           = -lxml2
# libs_static    = -lxml2 -lz -lpthread -licucore -lm
#
#
#
t/00_diag.t ........ ok
t/alien_libxml2.t .. 1/?
# Failed test 'xs'
# at t/alien_libxml2.t line 26.
#   ExtUtils::CBuilder->compile failed
#     error building /Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-9/_alien/tmp/kP0Cen_2lx/test.o from '/Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-9/_alien/tmp/kP0Cen_2lx/test.c' at /Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/lib/5.16.0/ExtUtils/CBuilder/Base.pm line 177.
#     cc -I/Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/lib/5.16.0/darwin-2level/CORE -c -fno-common -DPERL_DARWIN -mmacosx-version-min=10.15 -fno-strict-aliasing -pipe -fstack-protector -I/usr/local/include -DPERL_USE_SAFE_PUTENV -O3 -o /Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-9/_alien/tmp/kP0Cen_2lx/test.o /Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-9/_alien/tmp/kP0Cen_2lx/test.c
#     /Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-9/_alien/tmp/kP0Cen_2lx/test.xs:5:10: fatal error: 'libxml/parser.h' file not found
#     #include <libxml/parser.h>
#              ^~~~~~~~~~~~~~~~~
#     1 error generated.
t/alien_libxml2.t .. Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/3 subtests

Test Summary Report
-------------------
t/alien_libxml2.t (Wstat: 256 Tests: 3 Failed: 1)
  Failed test:  2
  Non-zero exit status: 1
Files=2, Tests=4,  0 wallclock secs ( 0.02 usr  0.00 sys +  0.42 cusr  0.12 csys =  0.56 CPU)
Result: FAIL
Failed 1/2 test programs. 1/4 subtests failed.
make: *** [test_dynamic] Error 1
  PLICEASE/Alien-Libxml2-0.16.tar.gz
  /usr/bin/make test -- NOT OK
//hint// to see the cpan-testers results for installing this module, try:
  reports PLICEASE/Alien-Libxml2-0.16.tar.gz
```

Unfortunately I get the same issue.

```
.cpan/build/Alien-Libxml2-0.16-1                                                                                                    ⍉
▶ make clean && perl Makefile.PL && make && make test
rm -f \
	  Libxml2.bso Libxml2.def \
	  Libxml2.exp Libxml2.x \
	   blib/arch/auto/Alien/Libxml2/extralibs.all \
	  blib/arch/auto/Alien/Libxml2/extralibs.ld Makefile.aperl \
	  *.a *.o \
	  *perl.core MYMETA.json \
	  MYMETA.yml blibdirs.ts \
	  core core.*perl.*.? \
	  core.[0-9] core.[0-9][0-9] \
	  core.[0-9][0-9][0-9] core.[0-9][0-9][0-9][0-9] \
	  core.[0-9][0-9][0-9][0-9][0-9] libLibxml2.def \
	  mon.out perl \
	  perl perl.exe \
	  perlmain.c pm_to_blib \
	  pm_to_blib.ts so_locations \
	  tmon.out
rm -rf \
	  blib
mv Makefile Makefile.old > /dev/null 2>&1
rm -rf _alien
Alien::Build::Plugin::PkgConfig::Negotiate> Using PkgConfig plugin: PkgConfig::CommandLine
Alien::Build::CommandSequence> + xml2-config --version
Alien::Build::CommandSequence> [output consumed by Alien::Build recipe]
Alien::Build::CommandSequence> + xml2-config --cflags
Alien::Build::CommandSequence> [output consumed by Alien::Build recipe]
Alien::Build::CommandSequence> + xml2-config --libs
Alien::Build::CommandSequence> [output consumed by Alien::Build recipe]
Checking if your kit is complete...
Looks good
Generating a Unix-style Makefile
Writing Makefile for Alien::Libxml2
Writing MYMETA.yml and MYMETA.json
cp lib/Alien/Libxml2.pm blib/lib/Alien/Libxml2.pm
"/Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/bin/perl" -MAlien::Build::MM=cmd -e prefix site /Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/lib/5.16.0/darwin-2level /Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/lib/site_perl/5.16.0/darwin-2level
Alien::Build::Plugin::PkgConfig::Negotiate> Using PkgConfig plugin: PkgConfig::CommandLine
main> prefix /Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/lib/site_perl/5.16.0/darwin-2level/auto/share/dist/Alien-Libxml2
"/Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/bin/perl" -MAlien::Build::MM=cmd -e version 0.16
Alien::Build::Plugin::PkgConfig::Negotiate> Using PkgConfig plugin: PkgConfig::CommandLine
"/Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/bin/perl" -MAlien::Build::MM=cmd -e download
Alien::Build::Plugin::PkgConfig::Negotiate> Using PkgConfig plugin: PkgConfig::CommandLine
"/Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/bin/perl" -MAlien::Build::MM=cmd -e build
Alien::Build::Plugin::PkgConfig::Negotiate> Using PkgConfig plugin: PkgConfig::CommandLine
Alien::Build::CommandSequence> + pkg-config --exists libxml-2.0
Alien::Build::CommandSequence> + pkg-config --cflags libxml-2.0
Alien::Build::CommandSequence> [output consumed by Alien::Build recipe]
Alien::Build::CommandSequence> + pkg-config --libs libxml-2.0
Alien::Build::CommandSequence> [output consumed by Alien::Build recipe]
Alien::Build::CommandSequence> + pkg-config --modversion libxml-2.0
Alien::Build::CommandSequence> [output consumed by Alien::Build recipe]
Alien::Build::CommandSequence> + pkg-config --static --cflags libxml-2.0
Alien::Build::CommandSequence> [output consumed by Alien::Build recipe]
Alien::Build::CommandSequence> + pkg-config --static --libs libxml-2.0
Alien::Build::CommandSequence> [output consumed by Alien::Build recipe]
Alien::Build::Plugin::Core::Legacy> adding legacy hash to config
Alien::Build::Plugin::Core::Gather> mkdir -p /Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-1/blib/lib/auto/share/dist/Alien-Libxml2/_alien
Manifying 1 pod document
"/Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/bin/perl" -MAlien::Build::MM=cmd -e test
Alien::Build::Plugin::PkgConfig::Negotiate> Using PkgConfig plugin: PkgConfig::CommandLine
PERL_DL_NONLAZY=1 "/Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
t/00_diag.t ........ 1/? #
#
#
# HARNESS_ACTIVE=1
# HARNESS_VERSION=3.42
# LANG=en_GB.UTF-8
# LC_TERMINAL=iTerm2
# LC_TERMINAL_VERSION=3.3.12
# PERL5LIB=/Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-1/blib/lib:/Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-1/blib/arch
# PERLBREW_HOME=/Users/michaelgrundie/.perlbrew
# PERLBREW_MANPATH=/Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/man
# PERLBREW_PATH=/Users/michaelgrundie/perl5/perlbrew/bin:/Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/bin
# PERLBREW_PERL=perl-5.16.0
# PERLBREW_ROOT=/Users/michaelgrundie/perl5/perlbrew
# PERLBREW_SHELLRC_VERSION=0.88
# PERLBREW_VERSION=0.88
# PERL_DL_NONLAZY=1
# PERL_USE_UNSAFE_INC=1
# SHELL=/bin/zsh
#
#
#
# PERL5LIB path
# /Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-1/blib/lib
# /Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-1/blib/arch
#
#
#
# perl                                     5.016000
# Alien::Base                              2.33
# Alien::Build                             2.33
# Alien::Build::MM                         2.33
# Alien::Build::Plugin::Build::SearchDep   2.33
# Alien::Build::Plugin::Prefer::BadVersion 2.33
# Alien::Build::Plugin::Probe::Vcpkg       2.33
# ExtUtils::CBuilder                       0.280206
# ExtUtils::MakeMaker                      7.48
# Test2::V0                                0.000136
# Test::Alien                              2.33
#
#
#
# version        = 2.9.4
# cflags         =
# cflags_static  =
# libs           = -lxml2
# libs_static    = -lxml2 -lz -lpthread -licucore -lm
#
#
#
t/00_diag.t ........ ok
t/alien_libxml2.t .. 1/?
# Failed test 'xs'
# at t/alien_libxml2.t line 26.
#   ExtUtils::CBuilder->compile failed
#     error building /Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-1/_alien/tmp/NTk4ly_305/test.o from '/Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-1/_alien/tmp/NTk4ly_305/test.c' at /Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/lib/5.16.0/ExtUtils/CBuilder/Base.pm line 177.
#     cc -I/Users/michaelgrundie/perl5/perlbrew/perls/perl-5.16.0/lib/5.16.0/darwin-2level/CORE -c -fno-common -DPERL_DARWIN -mmacosx-version-min=10.15 -fno-strict-aliasing -pipe -fstack-protector -I/usr/local/include -DPERL_USE_SAFE_PUTENV -O3 -o /Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-1/_alien/tmp/NTk4ly_305/test.o /Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-1/_alien/tmp/NTk4ly_305/test.c
#     /Users/michaelgrundie/.cpan/build/Alien-Libxml2-0.16-1/_alien/tmp/NTk4ly_305/test.xs:5:10: fatal error: 'libxml/parser.h' file not found
#     #include <libxml/parser.h>
#              ^~~~~~~~~~~~~~~~~
#     1 error generated.
t/alien_libxml2.t .. Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/3 subtests

Test Summary Report
-------------------
t/alien_libxml2.t (Wstat: 256 Tests: 3 Failed: 1)
  Failed test:  2
  Non-zero exit status: 1
Files=2, Tests=4,  0 wallclock secs ( 0.01 usr  0.01 sys +  0.44 cusr  0.13 csys =  0.59 CPU)
Result: FAIL
Failed 1/2 test programs. 1/4 subtests failed.
make: *** [test_dynamic] Error 1
```

--- a/Changes
+++ b/Changes
@@ -1,6 +1,7 @@
 Revision history for {{$dist->name}}
 
 {{$NEXT}}
+  - Bump required version of AB for bug fixes (gh#23, gh#24)
 
 0.16      2020-04-22 03:50:16 -0600
   - Fix mixed probe on macOS (gh#20)

--- a/README.md
+++ b/README.md
@@ -1,4 +1,4 @@
-# Alien::Libxml2 [![Build Status](https://travis-ci.org/PerlAlien/Alien-Libxml2.svg)](http://travis-ci.org/PerlAlien/Alien-Libxml2) ![macos-share](https://github.com/PerlAlien/Alien-Libxml2/workflows/macos-share/badge.svg) ![macos-system](https://github.com/PerlAlien/Alien-Libxml2/workflows/macos-system/badge.svg) ![windows-share](https://github.com/PerlAlien/Alien-Libxml2/workflows/windows-share/badge.svg) ![windows-system](https://github.com/PerlAlien/Alien-Libxml2/workflows/windows-system/badge.svg)
+# Alien::Libxml2 [![Build Status](https://api.travis-ci.com/PerlAlien/Alien-Libxml2.svg?branch=master)](https://travis-ci.com/github/PerlAlien/Alien-Libxml2) ![macos-share](https://github.com/PerlAlien/Alien-Libxml2/workflows/macos-share/badge.svg) ![macos-system](https://github.com/PerlAlien/Alien-Libxml2/workflows/macos-system/badge.svg) ![windows-share](https://github.com/PerlAlien/Alien-Libxml2/workflows/windows-share/badge.svg) ![windows-system](https://github.com/PerlAlien/Alien-Libxml2/workflows/windows-system/badge.svg)
 
 Install the C libxml2 library on your system
 

--- a/alienfile
+++ b/alienfile
@@ -1,7 +1,7 @@
 use alienfile;
 use Config;
 
-configure { requires 'Alien::Build' => '2.12' };
+configure { requires 'Alien::Build' => '2.37' };
 
 my $test_program = <<'EOF';
 #include <stdio.h>

--- a/dist.ini
+++ b/dist.ini
@@ -15,6 +15,7 @@ type = ffi
 [@Author::Plicease]
 :version      = 2.44
 travis_status = 1
+travis_com    = 1
 release_tests = 1
 installer     = Author::Plicease::MakeMaker
 irc           = irc://irc.perl.org/#native
@@ -46,16 +47,16 @@ remove = warnings
 remove = base
 
 [Prereqs]
-Alien::Base = 2.12
+Alien::Base = 2.37
 
 [Prereqs / ConfigureRequires]
 -phase = configure
-Alien::Build = 2.12
-Alien::Build::MM = 2.12
+Alien::Build = 2.37
+Alien::Build::MM = 2.37
 
 [Prereqs / BuildRequires]
--phase = configure
-Alien::Build = 2.12
+-phase = build
+Alien::Build = 2.37
 
 [Author::Plicease::Upload]
 cpan = 1

